rasa/core/policies/ted_policy.py

```diff
@@ -196,6 +213,7 @@ def __init__(
         priority: int = DEFAULT_POLICY_PRIORITY,
         max_history: Optional[int] = None,
         model: Optional[RasaModel] = None,
+        zero_features: Optional[Dict[Text, List["Features"]]] = None,
```

Comment:
is this rather `zero_example`? It contains `zero_features` for several attributes?

Reply:
No, it is not zero_example as it is just a dict of, for example, `"intent" -> [<features>]`. It does not contain the sub attributes like "sequence" in a second dict. However, the features itself have the type.

Reply:
ok, but let's update name `_features` here as in previous comment